### PR TITLE
Adds LIC3 issue #4

### LINKED ISSUE #4: Write function checking LIC 3

### PR DESCRIPTION
**Pull request by ayubatif**
Resolves #4 
Added LIC3 function as well as test code.

Included changes:

- LIC3 function
- testLIC3 function

### CODE REVIEW

**Comment by ayubatif** [comment]
undefined variable names in LIC3 need to be fixed ASAP.

File: lab1/src/test/java/lab1/TestDecide.java [hunk 1]
```diff
@@ -77,6 +77,28 @@ public void testLIC0() {
 	}
 	
 
+    @Test
```
Comment (ayubatif) [inline lab1/src/test/java/lab1/TestDecide.java]: Add javadoc explaining test returns

**Review by ayubatif — COMMENTED** [review]
Tests run properly but more documentation helps

File: lab1/src/test/java/lab1/TestDecide.java (hunk 1 above)
Comment (ayubatif) [reply lab1/src/test/java/lab1/TestDecide.java]: Done

**Review by evkade — APPROVED** [review]
Looks great in my opinion!

**Pull request merged**